src/dataExplorer/components/Results.tsx

```diff
@@ -25,31 +25,39 @@ import {FluxResult} from 'src/types/flows'
 
 import './Results.scss'
 import {isFlagEnabled} from 'src/shared/utils/featureFlag'
+import {bytesFormatter} from 'src/shared/copy/notifications'
 
 // simplified version migrated from src/flows/pipes/Table/view.tsx
 const QueryStat: FC = () => {
   const {result} = useContext(ResultsContext)
 
   const tableColumn = result?.parsed?.table?.getColumn('table') || []
-  const lastTableValue = tableColumn[tableColumn.length - 1] || -1
+  const lastTableValue = tableColumn[tableColumn.length - 1]
 
   let tableNum = 0
 
   if (typeof lastTableValue === 'string') {
     tableNum = parseInt(lastTableValue) + 1
   } else if (typeof lastTableValue === 'boolean') {
-    tableNum = lastTableValue ? 1 : 0
-  } else {
-    // number
+    console.error('Cannot extract tableId. Check parsed csv output.')
```

Comment:
I wasn't sure whether to log an event, or log an error to the console -- since most of our [internally filed] issue tickets include a snapshot of the browser console. So I'm also on the fence. 🤷🏼